composer.json

```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```

Comment:
Hey guys, FWIW I've moved this package to HipsterJazzbo/Landlord and made it 5.2 only.